catalog/MDCCatalog/MDCMenuViewController.swift

```diff
@@ -78,6 +101,9 @@ class MDCMenuViewController: UITableViewController {
                                                                    named: "Themes")
         navController.pushViewController(presentingViewController, animated: true)
       })
+    case 2:
```

Comment:
I think this could be a `break`

Reply:
Awesome! I didn't see any "breaks" in our Swift code so I didn't know they were allowed, thanks!

Reply:
Enums might be the best part of Swift 🙏

Reply:
enums?